src/kernel/arch/x86/32bit/arch.zig

```diff
@@ -419,12 +391,14 @@ pub fn getDateTime() DateTime {
 ///                                         paging.
 ///
 pub fn init(mem_profile: *const MemProfile) void {
-    gdt.init();
-    idt.init();
+    gdt_common.init(gdt_32.Tss, &gdt_32.tss_entry);
+    idt_common.init(idt_32.IdtEntry, &idt_32.table);
+
+    _ = interrupts;
```

Comment:
This line is needed to reference the `getInterruptStub()` function, unless there is a better way of doing it.

Reply:
Comment here would be good too.